src/index.html

```diff
@@ -17,16 +17,25 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="form-field">
+      <form data-qa="big">
+        <input
+          type="text"
+          data-qa="keypress"
```

Comment:
The `data-qa` attribute for the big search input should be updated to `big-search-input` to match the task requirements.